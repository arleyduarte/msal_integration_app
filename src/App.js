import React from 'react';
import {View, Text} from 'react-native';

const App: () => React$Node = () => {
  return (
    <View>
      <Text>Hola</Text>
    </View>
  );
};

export default App;
