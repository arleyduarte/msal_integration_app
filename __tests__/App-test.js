/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../src/App';
import { shallow } from 'enzyme';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<App />);
});

// Using Jest + Enzyme
describe('<App />', () => {
  it('renders correctly, test using Jest + Enzyme', () => {
    expect(shallow(<App/>)).toMatchSnapshot();
  });
});