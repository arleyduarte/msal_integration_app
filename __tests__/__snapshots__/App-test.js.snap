// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly, test using Jest + Enzyme 1`] = `
<View>
  <Text>
    Hola
  </Text>
</View>
`;
